src/Pos/DB/Block/Block.hs

```diff
@@ -244,8 +237,8 @@ putBi
     => ByteString -> v -> m ()
 putBi k v = rocksPutBi k v =<< getBlockIndexDB
 
-delete :: (MonadDB m) => ByteString -> m ()
-delete k = rocksDelete k =<< getBlockIndexDB
+-- delete :: (MonadDB m) => ByteString -> m ()
+-- delete k = rocksDelete k =<< getBlockIndexDB
```

Comment:
Don't forget to delete these comments in the future when they're not needed anymore.